### DISCUSSION TITLE
Login by another device doesn't work on linux (fedora)

### DISCUSSION THREAD

**Discussion by SegUun** [dani-garcia/vaultwarden#4379] [Q&A]
After update docker image on my server bitwarden client showing me this message. (When im trying to login with another device)
![Снимок экрана от 2024-02-23 15-12-42](https://github.com/dani-garcia/vaultwarden/assets/87121302/98626c19-37cd-431d-aafc-3950c4986642)

**Comment by BlackDex** [comment]
You probably need to clear your browser cache.

**Reply by SegUun** [reply]
In browser all ok. This error from desktop client
